[2025-03-10T17:42:50.871+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: project2.import_csv manual__2025-03-10T17:42:49.966216+00:00 [queued]>
[2025-03-10T17:42:50.878+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: project2.import_csv manual__2025-03-10T17:42:49.966216+00:00 [queued]>
[2025-03-10T17:42:50.879+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-03-10T17:42:50.879+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2025-03-10T17:42:50.879+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-03-10T17:42:50.890+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): import_csv> on 2025-03-10 17:42:49.966216+00:00
[2025-03-10T17:42:50.894+0000] {standard_task_runner.py:55} INFO - Started process 6908 to run task
[2025-03-10T17:42:50.897+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'project2', 'import_csv', 'manual__2025-03-10T17:42:49.966216+00:00', '--job-id', '126', '--raw', '--subdir', 'DAGS_FOLDER/project2.py', '--cfg-path', '/tmp/tmpqnqcdu2_']
[2025-03-10T17:42:50.897+0000] {standard_task_runner.py:83} INFO - Job 126: Subtask import_csv
[2025-03-10T17:42:50.949+0000] {task_command.py:388} INFO - Running <TaskInstance: project2.import_csv manual__2025-03-10T17:42:49.966216+00:00 [running]> on host 9f269e3d46f6
[2025-03-10T17:42:51.006+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=project2
AIRFLOW_CTX_TASK_ID=import_csv
AIRFLOW_CTX_EXECUTION_DATE=2025-03-10T17:42:49.966216+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-03-10T17:42:49.966216+00:00
[2025-03-10T17:42:51.006+0000] {sql.py:254} INFO - Executing: 
        COPY staging_hr FROM '/tmp/hr_data1.csv' 
        DELIMITER ',' CSV HEADER;
        
[2025-03-10T17:42:51.014+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2025-03-10T17:42:51.197+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2025-03-10T17:42:51.215+0000] {sql.py:375} INFO - Running statement: 
        COPY staging_hr FROM '/tmp/hr_data1.csv' 
        DELIMITER ',' CSV HEADER;
        , parameters: None
[2025-03-10T17:42:51.218+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 266, in execute
    **extra_kwargs,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 349, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 380, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.BadCopyFileFormat: extra data after last expected column
CONTEXT:  COPY staging_hr, line 2: "41,Yes,Travel_Rarely,1102,Sales,1,2,Life Sciences,1,1,2,Female,94,3,2,Sales Executive,4,Single,5993,..."

[2025-03-10T17:42:51.226+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=project2, task_id=import_csv, execution_date=20250310T174249, start_date=20250310T174250, end_date=20250310T174251
[2025-03-10T17:42:51.236+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 126 for task import_csv (extra data after last expected column
CONTEXT:  COPY staging_hr, line 2: "41,Yes,Travel_Rarely,1102,Sales,1,2,Life Sciences,1,1,2,Female,94,3,2,Sales Executive,4,Single,5993,..."
; 6908)
[2025-03-10T17:42:51.269+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2025-03-10T17:42:51.293+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
